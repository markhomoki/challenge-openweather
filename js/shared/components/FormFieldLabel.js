// @flow

import * as React from 'react';

type Props = {
	children: string,
};

export default class FormFieldLabel extends React.PureComponent<Props> {

	render() {
		return <span className="form-field-label">{this.props.children}</span>;
	}

}
