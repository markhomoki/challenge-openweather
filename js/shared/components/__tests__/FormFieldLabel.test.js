import React from 'react';
import renderer from 'react-test-renderer';
import FormFieldLabel from '../FormFieldLabel';

describe('FormFieldLabel', () => {
	it('renders correctly', () => {
		const tree = renderer.create(<FormFieldLabel />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
