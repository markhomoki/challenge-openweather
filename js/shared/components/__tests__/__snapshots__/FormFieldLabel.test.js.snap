// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormFieldLabel renders correctly 1`] = `
<span
  className="form-field-label"
/>
`;
